recover_affected_workloads.md

```diff
@@ -21,12 +21,17 @@ For example, in the case where the node fails to update its status to the
 masters the node condition will be set to `type:Ready status:Unknown`.
 
 In order to gain the workload back, reschedule it and start it all over, the
-control plane needs to know that the running node is gone and the only way to achieve
-that is by deleting the node object, which will trigger [scheduling the pods for
-deletion][2]
+control plane needs to know that the running node is gone and there are two ways to do it:
+
+- deleting the node object, which will trigger [scheduling the pods for deletion][2]
+- adding an [`out-of-service` taint][4] on the Node, which triggers pods on the node
+to be forcefully deleted if there are no matching tolerations on the pods.
+Persistent volumes attached to the shutdown node will be detached, and new pods
+will be created successfully on a different running node. This is the "right" way to do it.
+
+Currently, the "deletion strategy" is still supported by our [fence-agents-remediation](https://github.com/medik8s/fence-agents-remediation)
+operator (FAR), while both FAR and [self-node-remediation](/remediation/self-node-remediation/self-node-remediation/) (SNR) operator support the `out-of-service` taint.
```

Comment:
Over here it would be nice to add the automatic strategy while changing `the "deletion strategy"` -> `the "resource deletion strategy"`